smqtk_detection/impls/detect_image_objects/resnet_frcnn.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+
+import torch
+import torchvision.models as models
+from torchvision import transforms
+from torchvision.ops import boxes as box_ops
+import torch.nn.functional as F
+from torchvision.models.detection.roi_heads import RoIHeads
+
+from typing import Tuple, Iterable, Dict, Hashable, List
+
+from smqtk_detection.interfaces.detect_image_objects import DetectImageObjects
+from smqtk_detection.utils.bbox import AxisAlignedBoundingBox
+
+import warnings
+warnings.filterwarnings("ignore", category=UserWarning)
+
+
+class ResNetFRCNN(DetectImageObjects):
```

Comment:
The parent-class method
```
@classmethod
def is_usable() -> bool:
    ...
```
will need to be implemented. This will have to check that required things (e.g. the torch/torchvision imports) are correctly satisfied. See the `Pluggable` interface in smqtk-core for where this method is defined and documented.